### ISSUE TITLE
Table line not respecting table height with multiple tables across pages

### ISSUE THREAD

**Issue by rarysson** [simonbengtsson/jsPDF-AutoTable#805]
Hi, currently to create borders on a table I'm using the properties: tableLineWidth and tableLineColor. It works fine on multiple tables on the same page, but if I have multiple tables and one of the tables grow and it goes to another page, the line on the second table doesn't cover properly. As you can see on the image below:

![Screenshot_2021-05-28 random-1 pdf](https://user-images.githubusercontent.com/22689195/120035253-a78deb80-bfd4-11eb-8238-f868bc49eb8f.png)

It's a bug or I'm using something wrong? if you want to see the final PDF you can see below:

[pdf.pdf](https://github.com/simonbengtsson/jsPDF-AutoTable/files/6563044/pdf.pdf)

I was trying to create a jsfiddle to reproduce the bug, but I couldn't set up properly, but you can find the code here: [code](https://gist.github.com/rarysson/b5fe2b219d1a4ba87d7bc61eb7cb7c31)

**Comment by simonbengtsson** [comment]
Interesting! Can you reproduce as a minimal example as well?

**Comment by rarysson** [comment]
Yeah, you can check out here: https://jsfiddle.net/scorpion147/bon351uv/1/

**Comment by simonbengtsson** [comment]
Thanks. Marked it as a bug now.

**Comment by etienne-cartong** [comment]
Same issue here ! Did you find a workaround ? 
Mine is to pick the right colors (alternate) to avoid to use table line..

**Comment by rarysson** [comment]
@etienne-cartong sadly no, my woraround was to not use the table line on the table that would break on multiple pages.
